pagopa/io-app
# feat: [IOCOM-457] Navigate to the attachment preview screen

## Short description
This PR allows you to navigate to the preview of an F24

### Preview

| attachment |
| --- |
| <video src="https://github.com/pagopa/io-app/assets/29163287/ef1e9540-645b-482a-bc7d-3c27b675fc39" width="250" /> |

## List of changes proposed in this pull request
- added `AttachmentPreviewScreen`: added `category` parameter (for next Mixpanel events)
- passed the `openAttachment` callback to `MessageF24` component as a prop

## How to test
Using `io-dev-api-server` generate a SEND message with multiple F24. Tap on an F24 and check that it is displayed correctly.
